source/GStreamerEMEUtils.h

```diff
@@ -34,6 +35,12 @@ struct BufferProtectionMetadata
     // vector of bytesOfClearData, bytesOfEncryptedData
     std::vector<std::pair<uint32_t, uint32_t>> subsamples;
     uint32_t initWithLast15{0};
+
+    // Encryption scheme
+    firebolt::rialto::CipherMode cipherMode{firebolt::rialto::CipherMode::UNKNOWN};
+    uint32_t encryptedBlocks{0};
```

Comment:
Is it better to keep the block names the same as that in the metadata so that its clearer? So CryptBlocks and skipBlocks.

Reply:
ok, I've renamed them.